src/index.html

```diff
@@ -22,6 +22,360 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="links nav__links">
```

Comment:
do you need links as separate BEM block?

Reply:
So i must remain only nav_links class? 
Because in css even list child (li > a) started from link

Reply:
you didn't use or needed class "links" but block link you could left